apps/ledger-live-desktop/tests/enum/AppInfos.ts

```diff
@@ -5,6 +5,7 @@ export class AppInfos {
     public readonly name: string,
     public readonly sendPattern: DeviceLabels[],
     public readonly receivePattern: DeviceLabels[],
+    public readonly lsPattern?: DeviceLabels[],
```

Comment:
I think we should rework this part to have sendPattern and receivePattern optional